src/CommunityToolkit.Maui/Behaviors/PlatformBehaviors/SelectAllText/SelectAllTextBehavior.android.cs

```diff
@@ -12,7 +12,10 @@ protected override void OnAttachedTo(InputView bindable, EditText platformView)
 	{
 		base.OnAttachedTo(bindable, platformView);
 
-		platformView.SetSelectAllOnFocus(true);
+		if (!string.IsNullOrEmpty(platformView.Text))
```

Comment:
this doesn't look right, if you do this and then the text isn't null or empty the effect will not apply. You need to implement something that will that in account

Reply:
You are right